Automattic/woocommerce-payments
# Spike: Make request to update order on merchant store asynchronous

Spike. Main PR in 1223-gh-Automattic/woopay

#### Changes proposed in this Pull Request









#### Testing instructions





* Automated tests should pass.
* Refer to platform checkout repo PR for detailed testing instructions.
* Make sure critical payment flows work as intended.

-------------------


**Post merge**
